### PR TITLE
Adding /BehrCraft command, usage/description, file cleanup.

### PR DESCRIPTION
**Pull request by Eutherin**
1. Added /BehrCraft and /bc alias, as well as reportlag 

2. Added usage/description to the following commands:
- Survival
- Hub
- Lag_Report

3. Performed File cleanup in `behrcraft/repo-link/Structure/Formatting and Script Addons/`

4. Improved mobscripts:
- slime puddle creator.
- player destroying slime puddle handling.
- sound effect to shield busting script.

### CODE REVIEW

**Review by BehrRiley — CHANGES_REQUESTED** [review]
confused about what `slime_puddle_creator` is made for; the suggested changes there only correct what's within the script as it stands if `<context.entity>` were valid, but it has no `events` key or a valid event header. See my comment on the script container itself for details there - but the `player_slime_block_break_handler` script container makes sense in what it was supposed to do with what we talked about in discord, i suggested the changes that would resolve what's wrong with that.

File: denizen_scripts/global/server/commands/play.dsc [hunk 1]
```diff
@@ -124,12 +124,27 @@ survival_command:
...
+BehrCraft_Command:
+  type: command
+  debug: false
+  description: Sends a player to the BehrCraft server
+  usage: /behrcraft
+  name: BehrCraft
+  alias:
+  - bc
+  script:
+  - adjust <player> send_to:behrcraft
```
[the remark this answers is not in the document]
Comment (Eutherin) [reply denizen_scripts/global/server/commands/play.dsc:150]: Gotcha sorry I missed that.

File: denizen_scripts/survival/repo-link/MobScripts/SlimeStuff.dsc [hunk 2]
```diff
@@ -34,9 +34,33 @@ slime_mob_handler:
...
       - determine <context.damager.mythicmob.level.mul[3]>
+
+slime_puddle_creator:
+  type: world
+  debug: false
+  script:
+  - if <context.entity.is_mythicmob.entity[SLIME1]>
+    - define Puddlesize <context.entity.size.div[2]>
+    - define Puddle_Location <player.location.find.surface_blocks.within[Puddlesize]>
+    - flag server blocks_to_remove.<[Puddle_Location].simple>:<[Puddle_Location].material>
```
[the remark this answers is not in the document]
Comment (Eutherin) [reply denizen_scripts/survival/repo-link/MobScripts/SlimeStuff.dsc:52]: I thought that there was a remove flag command after modifyblock but see now it's missing. Thank you.

**Comment by Eutherin** [comment]
> confused about what `slime_puddle_creator` is made for; the suggested changes there only correct what's within the script as it stands if `<context.entity>` were valid, but it has no `events` key or a valid event header.

Per some discord comments, the script is meant to be called up by a mythicmob, would that not provide a valid context.entity for the script? It seemed easier than looping the script indefinitely every x ticks.

**Comment by BehrRiley** [comment]
> `Per some discord comments, the script is meant to be called up by a mythicmob, would that not provide a valid context.entity for the script?`

no

> `It seemed lighter than looping the script indefinitely every x ticks, or say an 'on entity moves' script.`

I'm not sure which would be heavier; but the former would likely be heavy depending on the number of slimes pathfinding, which includes if a player sets up a mob farm. this also raises the concern for verifying they're only converting solid blocks they step on, as opposed to literally everything always, like a stairway to heaven.

File: denizen_scripts/global/server/commands/play.dsc (hunk 1 above)
Comment (BehrRiley) [inline denizen_scripts/global/server/commands/play.dsc:150]: new line at the end of the file is good here
```suggestion
  - adjust <player> send_to:behrcraft

```

File: denizen_scripts/global/server/commands/play.dsc [hunk 3]
```diff
@@ -124,12 +124,27 @@ survival_command:
...
   script:
   - adjust <player> send_to:hub1
+
+BehrCraft_Command:
+  type: command
+  debug: false
+  description: Sends a player to the BehrCraft server
+  usage: /behrcraft
+  name: BehrCraft
+  alias:
```
Comment (BehrRiley) [inline denizen_scripts/global/server/commands/play.dsc:147]: `alias` should be `aliases`
```suggestion
  aliases:
```

File: denizen_scripts/global/server/commands/report_lag.dsc [hunk 4]
```diff
@@ -2,6 +2,10 @@ report_lag:
   type: command
   name: report_lag
   debug: false
+  description: Files a lag report in discord.
+  usage: /lag_report
+  Alias:
```
Comment (BehrRiley) [inline denizen_scripts/global/server/commands/report_lag.dsc:7]: `alias` should be `aliases`
```suggestion
  aliases:
```

File: denizen_scripts/survival/repo-link/MobScripts/SlimeStuff.dsc [hunk 5]
```diff
@@ -34,9 +34,33 @@ slime_mob_handler:
...
+
+player_slime_block_break_handler:
+  type: world
+  debug: false
+  script:
+  on player breaks:slime_block
+  - if <server.has_flag[blocks_to_remove.<[context.location].simple>]>:
+    - modifyblock <[Puddle_Location]> <server.flag[blocks_to_remove.<[<[Puddle_Location]>].simple>].as_material>
+    - playeffect effect:BLOCK_CRACK at:<context.location> special_data:SLIME_BLOCK quantity:20
+    - playsound <context.location> sound:BLOCK_SLIME_BLOCK_BREAK
```
Comment (BehrRiley) [inline denizen_scripts/survival/repo-link/MobScripts/SlimeStuff.dsc:66]: world script containers use `events`, not `script`
there also should be indention after the data key
```suggestion
  events:
    on player breaks:slime_block
      - if <server.has_flag[blocks_to_remove.<[context.location].simple>]>:
        - modifyblock <[Puddle_Location]> <server.flag[blocks_to_remove.<[<[Puddle_Location]>].simple>].as_material>
        - playeffect effect:BLOCK_CRACK at:<context.location> special_data:SLIME_BLOCK quantity:20
        - playsound <context.location> sound:BLOCK_SLIME_BLOCK_BREAK
```

File: denizen_scripts/survival/repo-link/MobScripts/SlimeStuff.dsc [hunk 6]
```diff
@@ -34,9 +34,33 @@ slime_mob_handler:
...
+          - playsound <context.location> sound:ENTITY_ITEM_BREAK
           - take slot:<[slot]>
 #Item Durability Damage
         - else:
           - inventory adjust slot:<[slot]> durability:<[Durrbillty]>
 #Damage player
       - determine <context.damager.mythicmob.level.mul[3]>
+
+slime_puddle_creator:
+  type: world
```
Comment (BehrRiley) [inline denizen_scripts/survival/repo-link/MobScripts/SlimeStuff.dsc:46]: this should be a task for now
```suggestion
  type: task
```

**Review by BehrRiley — APPROVED** [review]

**Pull request merged**